test/services/SortStatementsServiceSpec.scala

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
1. Could you please use constants instead of hard coded values especially for numbers as it produces the magic number warning? Example as below
2. Please make sure that the max line length is 120 characters. There are lots of file more than 120 characters.

    val periodStartYear = 2017
    val periodStartMonth = 12
    val periodStartDay =28
    val periodEndYear = 2018
    val periodEndMonth =1
    val periodEndDay =1
    val fileSize = 500L
    val size = 99L

Reply:
@tyagikumarvineet created constants to replace hard coded values in the tests